### ISSUE TITLE
Autocomplete suggestions positioned in wrong position

### ISSUE THREAD

**Issue by Mushtaq123** [XamFormsExtended/Xfx.Controls#25]
Please fill out either the bug or feature request section and remove whatever section you are not using.

## Bug 

* Version Number of Control: Latest
* Device Tested on: iPhone  5s
* Simulator Tested on: iPhone 7 Plus - iOS 11.0

### Expected Behavior
**XfxComboBox** should show the suggestions above or below the entry based on its position. 
### Actual Behavior
**XfxComboBox** should not show the suggestions above the entry when I kept the **combobox** bottom of the screen.
### Steps to reproduce the Behavior
1. Keep the Combobox at the bottom of the screen
2. Type your entry in **combobox**. 

Find the Screenshot below:

<table>
<tr>
<td>
<img src="https://lh4.googleusercontent.com/2pRr6sf6q3ahJoK62BImNNrzhVE6eJEDxw1FzaiFbHpRDdjeXXKlJU24E6AJywI1LUCLF424d0ucab0=w1280-h647-rw"/>
</td>
</tr>
</table>

## Feature Request:
Please fill in what you would like

**Comment by ChaseFlorell** [comment]
@Mushtaq123 I cannot see the screenshot, could you please upload it to the issue directly (instead of linking)? Also, it would be helpful if you can give me a snippet of XML to test.

**Comment by Mushtaq123** [comment]
Hi Chase,

The following **xaml** code has been used in my application.

```xaml
<?xml version="1.0" encoding="utf-8"?>
<ContentPage xmlns="http://xamarin.com/schemas/2014/forms"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             xmlns:xfx="clr-namespace:Xfx;assembly=Xfx.Controls"
             xmlns:app="clr-namespace:Xfx.Controls.Example;assembly=Xfx.Controls.Example"
             x:Class="Xfx.Controls.Example.MainPage"
             Title="Custom Controls Example">
    <ContentPage.BindingContext>
        <app:MainPageModel />
    </ContentPage.BindingContext>
    <ContentPage.Resources>
        <ResourceDictionary>
            <Style TargetType="Label">
                <Setter Property="VerticalTextAlignment" Value="Center" />
            </Style>
        </ResourceDictionary>
    </ContentPage.Resources>
    <StackLayout Orientation="Vertical">
        <Grid RowSpacing="0" ColumnSpacing="0">
            <Grid.RowDefinitions>
                <RowDefinition Height="Auto" />
                <RowDefinition Height="Auto" />
                <RowDefinition Height="Auto" />
                <RowDefinition Height="*" />
            </Grid.RowDefinitions>
            <Grid.ColumnDefinitions>
                <ColumnDefinition Width="1*" />
                <ColumnDefinition Width="2*" />
            </Grid.ColumnDefinitions>
            <Label Grid.Row="1" Grid.Column="0" Text="Material Entry" />
            <xfx:XfxEntry Grid.Row="1"
                          Grid.Column="1"
                          Placeholder="Enter your name"
                          Text="{Binding Name}"
                          ErrorText="{Binding NameErrorText}" />

            <Label Grid.Row="2" Grid.Column="0" Text="No Floating Label" />
            <xfx:XfxEntry Grid.Row="2"
                          Grid.Column="1"
                          Placeholder="This hint won't float, and it's a password entry"
                          Text="{Binding Foo}"
                          ErrorText="{Binding FooErrorText}"
                          IsPassword="True"
                          FloatingHintEnabled="False" />
        </Grid>

        <StackLayout VerticalOptions="EndAndExpand">
            <xfx:XfxComboBox Focused="Email_OnFocused"
                             Unfocused="Email_OnUnfocused"
                             Placeholder="Enter your email address"
                             Text="{Binding EmailAddress}"
                             ItemsSource="{Binding EmailSuggestions}"
                             SortingAlgorithm="{Binding SortingAlgorithm}" />
        </StackLayout>
    </StackLayout>
</ContentPage>
```
Find the Screenshot in below link

<a href="https://github.com/Mushtaq123/ErrorBox/blob/master/xfx_ios_bug.png">https://github.com/Mushtaq123/ErrorBox/blob/master/xfx_ios_bug.png</a>
<table>
<tr>
<td>
<img src="https://raw.githubusercontent.com/Mushtaq123/ErrorBox/master/xfx_ios_bug.png" height="700px" width="auto"/>
</td>
</tr>
</table>

**Comment by ChaseFlorell** [comment]
I see your issue now and will see what I can do.
I can say that if it were working as expected, your suggestions would show off of the view and  not be visible. Can you try wrapping your StackLayout in a `ScrollView` and get back to me

**Comment by ChaseFlorell** [comment]
This sounds like a duplicate of #19.

**Comment by Mushtaq123** [comment]
Dear Chase,

I have tried with wrapping StackLayout in a ScrollView and the result is same as before informed. Kindly, help me to use this control with Xamarin forms

**Comment by Mushtaq123** [comment]
Dear Chase,

Kindly advice on the previous comment posted by me. I need this to be used in my applications.

**Comment by ChaseFlorell** [comment]
Closed as duplicate of #19
